inc/dbutils.class.php

```diff
@@ -704,7 +704,7 @@ public function getSonsOf($table, $IDf) {
 
       if (!is_array($sons)) {
          // IDs to be present in the final array
-         $sons[$IDf] = "$IDf";
```

Comment:
i need another pov for this one

Reply:
This does not change anything, in the end, we just use this array to the `exportArrayToDB` method, which I guess does not care about type hinting.

Reply:
is cache reset on update ?

Reply:
Cache is updated at the end of the method